inspirehep/modules/records/jsonschemas/records/hep.json

```diff
@@ -195,10 +195,8 @@
                         "type": "string"
                     },
                     "orcid": {
-                        "description": "ORCID Id when available",
                         "format": "orcid",
-                        "pattern": "\\d{4}-\\d{4}-\\d{4}-\\d{4}",
-                        "title": "ORCID ID",
```

Comment:
As long as you know what you are doing I'm ok with it, just making sure it's not an 'unwanted' change. Maybe add a small note on the commit message.